apps/platform/src/assets/config.json

```diff
@@ -7,28 +7,15 @@
   "appUrl": "https://biosimulations.org/",
   "apiUrl": "https://api.biosimulations.org/",
   "email": "info@biosimulations.org",
+  "docsUrl": "https://docs.biosimulations.org/",
   "tutorialsUrl": null,
   "privacyPolicyVersion": 1,
   "newIssueUrl": "https://github.com/biosimulations/biosimulations/issues/new/choose",
   "newPullUrl": "https://github.com/biosimulations/biosimulations/compare",
-  "platformAppUrl": "https://biosimulations.org/",
-  "platformApiUrl": "https://api.biosimulations.org/",
   "platformNewIssueUrl": "https://github.com/biosimulations/biosimulations/issues/new/choose",
   "platformNewPullUrl": "https://github.com/biosimulations/biosimulations/compare",
-  "dispatchAppUrl": "https://run.biosimulations.org/",
-  "dispatchAppRunsNewUrl": "https://run.biosimulations.org/runs/new",
-  "utilsConvertFileUrl": "https://run.biosimulations.org/utils/convert-file",
-  "utilsCreateProjectUrl": "https://run.biosimulations.org/utils/create-project",
-  "utilsValidateModelUrl": "https://run.biosimulations.org/utils/validate-model",
-  "utilsValidateSimulationUrl": "https://run.biosimulations.org/utils/validate-simulation",
-  "utilsValidateMetadataUrl": "https://run.biosimulations.org/utils/validate-metadata",
-  "utilsValidateProjectUrl": "https://run.biosimulations.org/utils/validate-project",
-  "utilsSuggestSimulatorUrl": "https://run.biosimulations.org/utils/suggest-simulator",
-  "dispatchApiUrl": "https://api.biosimulations.org/",
   "dispatchNewIssueUrl": "https://github.com/biosimulations/biosimulations/issues/new/choose",
   "dispatchNewPullUrl": "https://github.com/biosimulations/biosimulations/compare",
-  "simulatorsAppUrl": "https://biosimulators.org/",
-  "simulatorsAppBrowseSimulatorsUrl": "https://biosimulators.org/simulators",
   "simulatorsApiUrl": "https://api.biosimulators.org/",
```

Comment:
shouldn't you remove simulatorsApiUrl here? You removed it in the dispatch app config file.

Reply:
Yes thank you for the heads up!

Reply:
Thanks for the heads up! I have removed these fields and implemented them as public properties of the new shared component as such:

`public validateSimulatorUrl = 'https://docs.biosimulations.org';`
`public apiUrl = 'https://api.biosimulations.org';`
`public simulatorsApiUrl? = 'https://api.biosimulators.org';`
`public docsUrl = 'https://docs.biosimulations.org';`